src/Sign.Cli/Program.cs

```diff
@@ -44,6 +44,18 @@ internal static async Task<int> Main(string[] args)
 
                 AddEnvironmentPath(netfxDir);
 
+                string systemDirectoryPath = Environment.GetFolderPath(Environment.SpecialFolder.System);
+
+                // NavSip.dll has a dependency on this.
+                string vcRuntime140FilePath = Path.Combine(systemDirectoryPath, "vcruntime140.dll");
+
+                if (!File.Exists(vcRuntime140FilePath))
+                {
+                    Console.ForegroundColor = ConsoleColor.Yellow;
```

Comment:
Maybe this could be an a separate method? `WriteWarning`?

Reply:
OK.  I _really_ wanted to use `ILogger.LogWarning(...)`, but I couldn't see how to get an `ILogger` this early in startup and honor logging options and reuse the same logger throughout.